httpserver/http_connection.cpp

```diff
@@ -215,6 +215,15 @@ void connection_t::start() {
     read_request();
 }
 
+void connection_t::notify(const message_t& msg) {
+    BOOST_LOG_TRIVIAL(debug)
+        << "Processing message notification: " << msg.data;
```

Comment:
The debug level is only meant for testing, where I either generate short messages or hardcode them. I guess I could use hashes instead, but that would mean changing all other uses of the message data. This could probably be its own PR in the future.